File: scala-scalatest/src/main/scala/com/github/webservicetesting/restingtodo/TodoItem.scala
package com.github.webservicetesting.restingtodo

import spray.json.DefaultJsonProtocol

/**
 * Created by amit on 3/26/15.
 */
case class TodoItem(name: String, task:String)

object TodoItemJsonProtocol extends DefaultJsonProtocol {
  implicit val colorFormat = jsonFormat2(TodoItem)
}

File: java-junit/src/test/java/com/github/webservicetesting/TodoItem.java
package com.github.webservicetesting;

/**
 * Created by amit on 3/31/15.
 */
public class TodoItem {
    public final String name;
    public final String task;

    public TodoItem() { this ( null,null);}

    public TodoItem(String name, String task) {
        this.name = name;
        this.task = task;
    }
}
